### PR TITLE
Update bug_report.md

### PR DESCRIPTION
**Pull request by nicolasmondada**
Updated to consolidated bug report template

### CODE REVIEW

**Review by NSeydoux — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -6,54 +6,40 @@ labels: "bug"
...
-3. See error
-
-**CodeSandbox**
-
-<!--
-Adding a minimal reproduction of the bug makes it as easy as possible to get it fixed.
-To do so, fork the following CodeSandbox, and share the resulting link here:
-
-https://codesandbox.io/s/github/inrupt/solid-client-js/tree/master/.codesandbox/sandbox
+Please fill in as much of the template below as you're able. Feel free to delete any section you want to skip.
```
Comment (Vinnl) [inline .github/ISSUE_TEMPLATE/bug_report.md:12]: As in the other PR, this is good, but I'd still encourage people to fill in as much as they can.

```suggestion
Please fill in as much of the template below as you're able. Feel free to delete any section you don't think applies, but be aware that the more comprehensive your description, the easier it is to take it into account. 
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -6,54 +6,40 @@ labels: "bug"
...
-<!-- A clear and concise description of what you expected to happen. -->
+### Bug description
+<!-- A short description of what the problem is. -->
 
-**Log output**
+### To Reproduce
+1. 
+2. 
+3. 
+4.
```
Comment (Vinnl) [inline .github/ISSUE_TEMPLATE/bug_report.md:25]: Why did you remove the CodeSandbox? That's a major boon in reproducing and hence solving bugs...

```suggestion
4. 

**Minimal reproduction**

<!--
Adding a minimal reproduction of the bug makes it as easy as possible to get it fixed.
To do so, fork the following CodeSandbox, insert the minimal code that demonstrates the problem,
and share the resulting link here:
https://codesandbox.io/s/github/inrupt/solid-client-js/tree/master/.codesandbox/sandbox
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -6,54 +6,40 @@ labels: "bug"
...
 
-<!-- If applicable, add log output to help explain your problem. -->
+### Expected result
+<!-- A clear and concise description of what you expected to happen -->
 
-**Environment**
+### Actual result
+<!-- A description of what actually happened -->
 
-<!--
```
Comment (Vinnl) [inline .github/ISSUE_TEMPLATE/bug_report.md:43]: Is there a reason this is not in a comment, whereas all the other descriptions on the headings are?

**Review by Vinnl — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (nicolasmondada) [reply .github/ISSUE_TEMPLATE/bug_report.md:25]: We can add it back

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -6,54 +6,48 @@ labels: "bug"
...
-
+To do so, fork the following CodeSandbox, insert the minimal code that demonstrates the problem,
+and share the resulting link here:
 https://codesandbox.io/s/github/inrupt/solid-client-js/tree/master/.codesandbox/sandbox
--->
-
-**Expected behavior**
 
-<!-- A clear and concise description of what you expected to happen. -->
+### Expected result
```
Comment (Vinnl) [inline .github/ISSUE_TEMPLATE/bug_report.md:35]: Thanks. I did inject a syntax error in my suggestion, so here's the fix - sorry!

```suggestion
-->

### Expected result
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (nicolasmondada) [reply .github/ISSUE_TEMPLATE/bug_report.md:43]: None really, added as a comment

**Review by Vinnl — APPROVED** [review]
